ui/dashboards/src/utils/copy.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const TOOLTIP_COPY = {
```

Comment:
Maybe rename this to `TOOLTIP_TEXT` or `TOOLTIP_LABEL`? I wasn't clear what copy meant at first glance

i think having a centralized place here for the labels is good - it will make it easier if we ever want to support localization